src/components/SingleList.jsx

```diff
@@ -1,19 +1,102 @@
-import { useNavigate } from 'react-router-dom';
 import './SingleList.css';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import PushPinIcon from '@mui/icons-material/PushPin';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { MaterialListButton, MaterialButton } from './material-buttons';
+import { useAuth, deleteList } from '../api';
+import { ConfirmDialog } from './ConfirmDialog';
+
+export const deletionResponse = {
+	hard: `List deleted permanently.`,
+	soft: `List removed from user view.`,
+};
```

Comment:
Group PR review: no need to export this if it's just being used in this file.

Reply:
Very fair! I planned to use it in tests and thought that I could export the responses that should be visible on the screen